packages/ui/src/validators/components/ValidatorsList.tsx

```diff
@@ -19,8 +19,10 @@ export const ValidatorsList = ({ validators }: ValidatorsListProps) => {
         <ListHeader>Validator</ListHeader>
         <ListHeader>Verification</ListHeader>
         <ListHeader>State</ListHeader>
-        <ListHeader>Total Reward</ListHeader>
-        <ListHeader>APR</ListHeader>
+        <ListHeader>Own Stake</ListHeader>
+        <ListHeader>Total Stake</ListHeader>
+        <ListHeader>Expected Nom APR</ListHeader>
```

Comment:
I know it's not in the design but personally I could really use a tooltip on "Expected Nom APR" !

WDYT @dmtrjsg ?

Reply:
Tooltip is 100% warranted. 

Text: "This column shows the expected APR for nominators who are nominating funds for the chosen validator. The APR is subject to the amount staked and have a diminishing return for higher token amounts."